From c51bf45304812b4da010bdd3db1b1d59c44af4ee Mon Sep 17 00:00:00 2001
From: Uri Lublin <uril@redhat.com>
Date: Tue, 27 Apr 2021 18:37:09 +0300
Subject: [PATCH 2/4] Remove SPICE and QXL from x86_64-rh-devices.mak
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Uri Lublin (uril)
RH-MergeRequest: 3: Build qemu-kvm without SPICE and QXL
RH-Commit: [1/3] 8f6186049d22c3a6e1bfb1cf2bfe88a8fc8c2271
RH-Bugzilla: 1906168
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>
RH-Acked-by: Danilo de Paula <ddepaula@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
---
 default-configs/devices/x86_64-rh-devices.mak | 2 --
 1 file changed, 2 deletions(-)

diff --git a/default-configs/devices/x86_64-rh-devices.mak b/default-configs/devices/x86_64-rh-devices.mak
index 9f41400530..d09c138fc6 100644
--- a/default-configs/devices/x86_64-rh-devices.mak
+++ b/default-configs/devices/x86_64-rh-devices.mak
@@ -60,7 +60,6 @@ CONFIG_PFLASH_CFI01=y
 CONFIG_PVPANIC_ISA=y
 CONFIG_PXB=y
 CONFIG_Q35=y
-CONFIG_QXL=y
 CONFIG_RTL8139_PCI=y
 CONFIG_SCSI=y
 CONFIG_SERIAL=y
@@ -70,7 +69,6 @@ CONFIG_SEV=y
 CONFIG_SGA=y
 CONFIG_SMBIOS=y
 CONFIG_SMBUS_EEPROM=y
-CONFIG_SPICE=y
 CONFIG_TEST_DEVICES=y
 CONFIG_USB=y
 CONFIG_USB_EHCI=y
-- 
2.27.0

